encoding/masking-limiting-etc.md

```diff
@@ -22,9 +22,28 @@ This article will cover:
 
 Masking refers to a broad set of techniques used to merge multiple clips,
 usually one filtered clip merged with a source clip
-according to an overlay mask.
-A mask clip may contain any information
-generated from a pixel-wise operation.
+according to an overlay mask clip.
+A mask clip specifies the weight for each individual pixel
+according to which the two clips are merged;
+see [MaskedMerge](#std-MaskedMerge) for details.
+
+In practice,
+masks are usually used to protect details,
+texture, and/or edges
+from destructive filtering effects like smoothing;
+this is accomplished
+by masking the areas to protect,
+e.g. with an edgemask,
+and merging the filtered clip with the unfiltered clip
+according to the mask,
+such that the masked areas are taken from the unfiltered clip,
+and the unmasked areas are taken from the filtered clip.
+In effect,
+this applies the filtering only to the unmasked areas of the clip,
+leaving the masked details/edges intact.
+(In this context,
+“masked” means that the pixels in question are white in the mask clip.)
```

Comment:
This parenthesized explanation doesn't really help with understanding the internals because it's too unspecific. Either be more specific on how it works or leave it out entirely since the more detailed explanation is in line 55.